src/domains/transaction/utils.ts

```diff
@@ -14,6 +14,8 @@ export const evaluateFee = (fee: any) => {
 };
 
 export const hasSufficientFunds = ({ wallet, fee = 0, amount = 0 }: SufficientFundsParams) => {
+	if (wallet.balance().isEqualTo(0)) return false;
```

Comment:
```suggestion
	if (wallet.balance().isZero()) return false;
```
There's a helper method for checks against zero, as zero can be either positive or negative. I think `isEqualTo(0)` just check against `+0`, haven't tested though to be honest and I don't know under what circumstances zero becomes negative 🤷